packages/@aws-cdk/aws-codepipeline/lib/pipeline.ts

```diff
@@ -146,6 +146,13 @@ export interface PipelineProps {
    * @default - false (key rotation is disabled)
    */
   readonly enableKeyRotation?: boolean;
+
+  /**
+   * Generate a unique cross region support stack.
+   *
+   * @default - false (Use the same support stack for all pipelines in App)
+   */
+  readonly uniqueCrossRegionStackName?: boolean;
```

Comment:
Let's call this property `reuseCrossRegionSupportStacks` instead, and let's make the default `true`.